src/test/groovy/com/bullhornsdk/data/api/TestMockDataLoader.groovy

```diff
@@ -32,11 +32,7 @@ public class TestMockDataLoader {
 		Map<Class<? extends BullhornEntity>, Map<Integer, ? extends BullhornEntity>> entityData = mockDataLoader.getEntityTestData();
 
 		entityData.each { key,value ->
-            if (CustomObjectBase.class.isAssignableFrom(key)) {
-                assert value.size() == 1;
-            } else {
-                assert value.size() >= 20;
-            }
+            assert value.size() >= 1;
```

Comment:
Wouldn't we want to leave the if statements? Don't you want the `value.size() == 1` only if the CustomObject is assignable from the key?

Reply:
The assertion all that really checks is that the test data has appropriate length. The TXT files had 20 records for all entities but I did not add 20 records of each new entity